census/src/main/scala/grasshopper/census/search/CensusGeocode.scala

```diff
@@ -34,6 +33,86 @@ trait CensusGeocode {
     }
   }
 
+  // An alternative geocoding implementation, uses less Elasticsearch query and more filtering in code. Not using for now
```

Comment:
Do you think we'll end up using this function at some point?  If not, move it over to a dev branch somewhere?  If yes, can we rename the 2 `geocodeLine` functions to better describe what each does, and a scaladoc comment as to why you'd use one vs. the other?